crates/net/network/src/manager.rs

```diff
@@ -676,7 +676,7 @@ where
                             let total_active =
                                 this.num_active_peers.fetch_add(1, Ordering::Relaxed) + 1;
                             this.metrics.connected_peers.set(total_active as f64);
-                            info!(
+                            trace!(
```

Comment:
I'd rather go for debug level, because these logs are not as frequent as other network-related logs we already have on trace level in other crates.

Reply:
Also maybe combine it with the next debug log `Established peer enode`?

Reply:
debatable because of the intention of the log: `debug` as the name suggests is mostly for debugging, `trace` is for more detailed information about a particular action of the application

Reply:
merged the two, pending @mattsse

Reply:
In my head `trace` was always about even more hardcore debugging than `debug`, and mostly useful for core devs who want to step through the function behavior using these `trace` logs. While `debug` is also for regular users, who would like to know that, e.g. in the case, they established a connection with some peer but without too much implementation-specific details.

Reply:
both `trace` and `debug` are for debugging, imo we abuse `trace` a little and do not use `debug` when needed. I usually use debug when it can be read from the console but we don't want to have it in a normal flow, and trace if it is something that can only be read if it is in file (log too big or it is frequent (spammy)).